wdl/ExpansionHunter.wdl

```diff
@@ -142,3 +165,145 @@ task RunExpansionHunter {
         maxRetries: runtime_attr.max_retries
     }
 }
+
+task SplitVariantCatalog {
+    input {
+        File variant_catalog
+        Int batch_size
+        String? split_variant_catalog_docker
+        String? output_prefix
+        RuntimeAttr? runtime_attr_override
+    }
+
+    output {
+        Array[File] catalogs_json = glob("${output_prefix_}*.json")
+    }
+
+    String output_prefix_ =
+        if defined(output_prefix) then
+            select_first([output_prefix])
+        else
+            "output_"
+
+    String split_variant_catalog_docker_ =
+        if defined(split_variant_catalog_docker) then
+            select_first([split_variant_catalog_docker])
+        else
+            "python:slim-buster"
+
+    command <<<
+        set -euxo pipefail
+
+        python <<CODE
+        import json
+        import sys
+        import os
+        from pathlib import Path
+
+        filename = "~{variant_catalog}"
+        filename_without_ext = Path(filename).stem
+        output_prefix = "~{output_prefix_}"
+        i = 0
+        subset_counter = 0
+
+        def serialize():
+            print(f"subset counter: {subset_counter}")
+            with open(f"{output_prefix}{filename_without_ext}_{subset_counter}.json", "w") as f_out:
+                json.dump(subset_catalogs, f_out, indent=4)
+
+        with open(filename, "r") as f_in:
+            catalogs = json.load(f_in)
+
+            subset_catalogs = []
+            for catalog in catalogs:
+                subset_catalogs.append(catalog)
+                i += 1
+                if i == ~{batch_size}:
+                    serialize()
+                    i = 0
+                    subset_counter += 1
+                    subset_catalogs = []
+            serialize()
```

Comment:
I think this might get run even if `subset_catalogs` is empty, so you'd want to check that that's not the case.

Reply:
Good catch, thanks! I added a condition check, hopefully, that will prevent serialising an empty array.